.github/labeler.yml

```
@@ -0,0 +1,50 @@
+# Folder Rules
+'Changelog Required':
+- '!html/changelogs/*.yaml'
+
+Database:
+- SQL/*
+
+'🗺️ Mapping - Aurora':
+- maps/aurora/*
+
+'🗺️ Mapping - Away Ship/Away Site':
+- maps/away/*
+
+'🗺️ Mapping - Dungeon':
```

```suggestion
'🗺️ Mapping - Dungeons':
```
since ruins was chosen to remain plural then this should probably be plural as well.